client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -662,6 +663,10 @@
                     superdesk.intent('view', 'content');
                 };
 
+                $scope.isMediaType = function isMediaType(item) {
+                    return item && _.contains(['audio', 'video', 'picture'], item.type);
```

Comment:
You are right, will fix.

On Wed, Apr 8, 2015, 18:24 Petr Jašek <notifications@github.com> wrote:

> In client/app/scripts/superdesk-authoring/authoring.js
> <https://github.com/superdesk/superdesk/pull/339#discussion_r27951070>:
>
> > @@ -662,6 +663,10 @@
> >                      superdesk.intent('view', 'content');
> >                  };
> >
> > +                $scope.isMediaType = function isMediaType(item) {
> > +                    return item && _.contains(['audio', 'video', 'picture'], item.type);
>
> this won't change during item editing, so it could be like $scope.isMediaType
> = _.contains(.., item.type);
> there is no need to do this iteration in every digest. or those could be
> just one time watches in template
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/superdesk/superdesk/pull/339/files#r27951070>.
>